src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -31,11 +31,14 @@ const getSelectorLastPart = (selector: string): string => {
 };
 
 const appendSuffixToUrl = (url: string, suffix: string): string => {
-    if (endsWith(url, suffix) || endsWith(url, `${suffix}/`)) {
+    if (endsWith(url, suffix)) {
```

Comment:
This will not append issues to http://github.com/kleenex/tissues - maybe check as
```suggestion
    if (endsWith(url, `/${suffix}`)) {
```
although this doesn't handle http://github.com/magazine/issues (repo name is issues - [a few folks do this](https://github.com/search?p=4&q=issues&type=Repositories))

Reply:
Base on your second point... we should be always appending `/issues` to the urls as the place holder already reads: `https://github.com/owner/repo` (no issues in it). Up to the user to make sure they don't add the `/issues` suffix. Does this make sense?

Reply:
This make sense. I added this check to make it compatible with old bug filing UX. Since that's completely removed, we do not need it anymore.

Reply:
After a second thought, I think it might cause issue if the user has used the old bug filing feature. It would break for them if we always append '/issues'.

Reply:
Yeah, we should be backwards compatible with users with old settings

Reply:
Updated.
Using regex to match expected github url.